src/Symfony/Component/Security/composer.json

```diff
@@ -32,6 +32,12 @@
         "psr/log": "~1.0",
         "ircmaxell/password-compat": "~1.0"
     },
+    "conflict": {
+      "symfony/security-acl": "*",
+      "symfony/security-core": "*",
+      "symfony/security-csrf": "*",
```

Comment:
this one is wrong. ``security-csrf`` was new code, which does not exist in the 2.3 branch